lib/views/app.js

```diff
@@ -97,10 +98,33 @@ class AppView extends React.Component {
     this.scan({ clear: true });
   }
 
+  onItemSwiped(item) {
+    this.removeItem(item.props.id);
+  }
+
+  setItems(items) {
+    LayoutAnimation.configureNext(LayoutAnimation.Presets.easeInEaseOut);
+    this.setState({ items });
+  }
+
+  removeItem(id) {
+    debug('remove item', id);
+    var items = this.state.items;
+
+    for (let i = 0; i < items.length; i++) {
```

Comment:
Sure. Although I was wary of support, if this is being run inside an old Android JS env. Does babel cover these kind of APIs?